include/lsst/afw/image/Calib.h

```diff
@@ -51,6 +52,29 @@ namespace cameraGeom {
     class Detector;
 }
 namespace image {
+
+static double const JanskysPerABFlux = 3631.0;
+
+/// Compute AB magnitude from flux in Janskys
+inline double abMagFromFlux(double flux) {
+    return -2.5*std::log10(flux/JanskysPerABFlux);
+}
+
+/// Compute AB magnitude error from flux and flux error in Janskys
+inline double abMagErrFromFluxErr(double fluxErr, double flux) {
+    return std::abs(fluxErr/(-0.4*flux*std::log(10)));
+}
```

Comment:
Of course, "flux" doesn't imply "DN" either, and we've been quite happy to ignore that elsewhere in our code so far (well, perhaps because it *could* imply DN until now).

This is something we've been struggling with for the past week or so.  I'm not bothered by this as it is mostly because I don't expect it to stay that way forever.  But I also don't have a good long-term solution (I'd rather not take the step of embedding units as types in *all* of our interfaces).